.travis.yml

```diff
@@ -23,6 +23,9 @@ script:
 - git clone https://github.com/RunestoneInteractive/thinkcspy.git
 - cd ..
 - cd tests
+# Rebuild the grades just to make sure this works.
+- python run_tests.py --rebuildgrades
```

Comment:
We really shouldn't run rebuildgrades UNLESS we know we have intentionally changed something that changes how autograding works.  Otherwise we may be just hiding unintentional breakage by rebuilding.  If we do run rebuild grades then we should re-dump the database and update runestone_test.sql

Reply:
That makes sense. I still want to run the `--rebuildgrades` options just to make sure it runs, even if the output isn't used. Should I move this to be after the tests are run, then?

Reply:
Its fine to run it either before or after but just don't dump the database.

Reply:
I assume you mean that the code should not run [pg_dump](https://github.com/RunestoneInteractive/RunestoneServer/blob/master/tests/run_tests.py#L52)? If so, how do you want me to do this?

1. Add another command-line option that just run `make_clean_db_with_grades.py` but not `pg_dump`?
2. Do a `git checkout -- tests/runestone_test.sql` to revert any changes.
3. Run this after `python run_test.py`, so the changes don't matter.

The third choice seems simplest to me.